recipes/pytest-bandit/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pytest-bandit" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "40899d30f39af3e8242cb1fd3d086317ef6f9440f3e5370c28f7212b2b15bf28"
+
+build:
+  skip: true  # [py27]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - pytest >=3.5.0
+    - bandit >=1.4.0
+
+test:
+  imports:
+    - pytest_bandit.controller
+    - pytest_bandit.plugin
```

Comment:
You can test the pytest entrypoint by something like ` - "py.test --traceconfig | grep pytest_bandit-{{version}}"`, see e.g. https://github.com/conda-forge/pytest-regtest-feedstock/blob/master/recipe/meta.yaml#L28